rules/web/proxy_generic/proxy_hello_world_user_agent.yml

```
@@ -0,0 +1,24 @@
+title: Detect HTTP GET Request from User-Agent Hello-World/1.0
+id: 1712bafe-be05-4a0e-89d4-17a3ed151bf5
+status: experimental
+description: |
+  Detects HTTP GET requests originating from the 'Hello-World/1.0' user agent to common web ports.
+references:
+    - https://www.greynoise.io/blog/new-scraper-botnet-concentrated-in-taiwan
+    - https://viz.greynoise.io/tags/hello-world-scraper-botnet?days=30
+author: Joseph A. M.
+date: 2025-08-02
+tags:
+    - attack.reconnaissance
+    - attack.t1595
+logsource:
+    category: proxy
+detection:
+    selection:
+        c-useragent: 'Hello-World/1.0'
+        cs-method: 'GET'
+    condition: selection
+falsepositives:
+    - Legitimate network monitoring or vulnerability scanning tools that may use this generic user agent.
+    - Internal development or testing scripts. Consider filtering by source IP if this is expected from certain systems.
+level: low
```

```suggestion
level: medium
```
If this useragent is specific enough, we can maybe make this a medium lvl rule.